### ISSUE TITLE
Sort Filter - 1.3.1 Info and Relationships - The select component is missing a label

### ISSUE THREAD

**Issue by imogenroberts** [archesproject/arches#9996]
As a user reliant on the screen reader for navigation, use of components and building a mental image of the page, I expect each component to have an accessible label so that the screen reader can correctly communicate the names and purposes of all components in the page. 

**Linked Work Item:**
#9496  

**Expected Result:**
The component should have an accessible label so that when the user focuses on the sort filter dropdown, the screen reader should communicate to the user the name and function of the component.

**Actual Result:**
The sort filter does not have an accessible label, meaning that when the user focuses on the component, the screen reader reads out 'Read only, blank', which provides no information to the user about the component or what it does.

**Evidence:**

![Image](https://github.com/archesproject/arches/assets/77161085/d529fe8f-17ab-4d67-ae59-605e2895fc60)

**Comment by johnatawnclementawn** [comment]
Resolved in #10069

**Comment by imogenroberts** [comment]
RETEST: The select filter is no longer missing a label. The screen reader however still reads the Sort Filter as 'Edit, Read only, Blank'. This should be resolved with the addition of a label to the component with accessible alt text describing the name and purpose of the select component.

**Evidence:**
![image](https://github.com/archesproject/arches/assets/77161085/9e756b5f-fd5f-4adc-b454-ab7d3d43a8d9)

**Comment by imogenroberts** [comment]
RETEST: The sort filter is no longer missing its label and is therefore read out correctly by the screen reader.

**Evidence:**
![Sort Filter - 1 3 1 Info and Relationships FIXED ](https://github.com/archesproject/arches/assets/77161085/9d8296a2-ac0f-4f71-950b-ddc04cee2d2c)
